compiler/src/dotty/tools/dotc/typer/ErrorReporting.scala

```diff
@@ -125,7 +125,7 @@ object ErrorReporting {
     /** Explain info of symbol `sym` as a member of class `base`.
      *   @param  showLocation  if true also show sym's location.
      */
-    def infoString(sym: Symbol, base: Type, showLocation: Boolean): String =
+    def infoString(sym: Symbol, base: Type, showLocation: Boolean): String = atPhase(Phases.typerPhase) {
```

Comment:
I am a bit nervous about doing this phase switch for all possible messages. What if symbols get created or changed in an intermediate phase? What does Scala 2 do in this case?

Alternatively, we could also do the `atPhase(typerPhase)` just for the
```
          overrideError("cannot override because erased signatures conflict in repeated parameter")
```

Reply:
It should have been using `DoubleDefinition` which is clever enough. This happens in erasure in Scala 2, where it is slightly more natural. So now, for the `override` case, it `overrides nothing`, otherwise it is `Name clash between defined and inherited member`.